v3/src/components/case-table/case-table-component.tsx

```diff
@@ -27,30 +25,36 @@ const collisionDetection: CollisionDetection = (args) => {
 }
 registerTileCollisionDetection(kCaseTableIdBase, collisionDetection)
 
-export const CaseTableComponent = observer(function ({ tile }: ITileBaseProps) {
-  const env: ITileEnvironment = getEnv(tile)
-  const tableModel = tile?.content
-  const data = useDataSetContext()
-  // find the metadata that corresponds to this DataSet
-  const caseMetadata =
-    env.sharedModelManager
-      ?.getSharedModelsByType(kSharedCaseMetadataType)
-       .find((model: ISharedCaseMetadata) => {
-        return model.data?.id === data?.id
-      }) as ISharedCaseMetadata | undefined
-  if (!isCaseTableModel(tableModel)) return null
-
+export const CaseTableComponent = observer(function CaseTableComponent({ tile }: ITileBaseProps) {
   const instanceId = useNextInstanceId(kCaseTableIdBase)
   // pass in the instance id since the context hasn't been provided yet
   const { setNodeRef } = useTileDropOverlay(instanceId)
 
+  const tableModel: ICaseTableModel | undefined = isCaseTableModel(tile?.content) ? tile?.content : undefined
```

Comment:
`null` is conventional as a return value from React components that render nothing. For most other use cases I prefer `undefined`. For conditional testing, I personally prefer `!= null` because it is compact and catches both `null` and `undefined`.